editor/gui/editor_toaster.cpp

```diff
@@ -371,6 +371,20 @@ Control *EditorToaster::popup(Control *p_control, Severity p_severity, double p_
 	p_control->set_h_size_flags(SIZE_EXPAND_FILL);
 	hbox_container->add_child(p_control);
 
+	// Copy button.
+	if (p_time > 0.0) {
+		Button *copy_button = memnew(Button);
+		copy_button->set_flat(true);
+		copy_button->set_button_icon(get_editor_theme_icon(SNAME("ActionCopy")));
```

Comment:
This is done in theme changed when button is added to tree, so unnecessary to set it here.